pkg/transformer/inline.go

```diff
@@ -148,6 +148,15 @@ func (n *Inliner) processComponentPackageFiles(ctx context.Context, b *bpb.Clust
 					ocf.Url = "file://" + filepath.Join(filepath.Dir(shortFileUrl(compUrl)), shortFileUrl(objUrl))
 				}
 			}
+
+			// Do the same with the text files.
+			for _, ocf := range comp.GetSpec().GetRawTextFiles() {
+				compUrl := cf.GetUrl()
```

Comment:
Nit: 'compUrl' is the same every iteration, so it can be pulled out of this for loop, and from the loop on line 145, to line 143.

Reply:
Done.